assignments/closure.js

```diff
@@ -1,14 +1,22 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
-
-
+function multiplyNums(a, b) {
+  return a * b
+}
+function whatIsATimesB(a, b, cb) {
+  return `A times B is equal to ${cb(a, b)}.`
+}
+console.log(whatIsATimesB(10, 20, multiplyNums))
 // ==== Challenge 2: Create a counter function ====
```

Comment:
This is a great example of a callback and I'm glad you have a solid understanding of them. Unfortunately this is asking for an example of a closure. If you have the time, go back here and create a closure